.changeset/khaki-hairs-vanish.md

```diff
@@ -0,0 +1,8 @@
+---
+"fluid-framework": patch
+"@fluidframework/map": patch
+---
+
+Map API documentation updates
+
+Remaining top-level APIs within `@fluidframework/map` have had docs updated to reflect their sealed nature, i.e. they are not to be implemented externally to Fluid Framework.
```

Comment:
Note for @microsoft/fluid-cr-docs follow-up - does not block this PR:

For changes like this, I think the changeset should include a list of the APIs that were changed, ideally with a link to the API docs for each item.